src/main/java/xyz/deseteral/lebkuchenfm/domain/mappers/GenericCommandResponseDtoMapper.java

```diff
@@ -0,0 +1,20 @@
+package xyz.deseteral.lebkuchenfm.domain.mappers;
+
+import xyz.deseteral.lebkuchenfm.api.commands.TextIsNotACommandException;
+import xyz.deseteral.lebkuchenfm.domain.CommandProcessingResponse;
+import xyz.deseteral.lebkuchenfm.domain.dto.GenericCommandResponseDto;
+import xyz.deseteral.lebkuchenfm.services.commands.NoSuchCommandException;
+
+public class GenericCommandResponseDtoMapper {
+    public static GenericCommandResponseDto from(CommandProcessingResponse processingResponse) {
```

Comment:
Skoro to DTO to sam obiekt DTO powinien umieć taką transformację wykonać? Mappery tego typu często stają się środkiem wszechświata, który finalnie ciężko opanować.

Reply:
Done